dataflow/src/main/java/org/checkerframework/dataflow/expression/JavaExpression.java

```diff
@@ -205,17 +225,17 @@ public String toStringDebug() {
      * Returns the internal representation (as {@link FieldAccess}) of a {@link FieldAccessNode}.
      * The result may contain {@link Unknown} as receiver.
      *
+     * @param node the FieldAccessNode to convert to a JavaExpression
      * @return the internal representation (as {@link FieldAccess}) of a {@link FieldAccessNode}.
      *     Can contain {@link Unknown} as receiver.
      */
-    public static FieldAccess fromNodeFieldAccess(
-            AnnotationProvider provider, FieldAccessNode node) {
+    public static FieldAccess fromNodeFieldAccess(FieldAccessNode node) {
```

Comment:
All the public method signatures that have changed in this file should be listed in the change log.